app/src/main/java/tech/relaycorp/gateway/data/model/LocalEndpoint.kt

```diff
@@ -1,12 +1,15 @@
 package tech.relaycorp.gateway.data.model
 
 import androidx.room.Entity
+import androidx.room.Index
+import androidx.room.PrimaryKey
 
 @Entity(
     tableName = "Endpoint",
-    primaryKeys = ["address", "applicationId"]
+    indices = [Index(value = ["applicationId"], unique = false)]
```

Comment:
Why isn't it a PK anymore? And why `unique=false`?

Reply:
If there's only one PK, we can use the annotation `@PrimaryKey` straight on the `address` field. And I marked the `applicationId` and non-unique because I believe we discussed that one app could have multiple addresses.

Reply:
> If there's only one PK, we can use the annotation @PrimaryKey straight on the address field

Indeed, there'd only be one PK.

> And I marked the applicationId and non-unique because I believe we discussed that one app could have multiple addresses

My bad! Yes, that's correct.